src/app/layout.tsx

```diff
@@ -1,21 +1,21 @@
-import './globals.css'
-import { Inter } from 'next/font/google'
+import "./globals.css";
+import { Inter } from "next/font/google";
 
-const inter = Inter({ subsets: ['latin'] })
+const inter = Inter({ subsets: ["latin"] });
 
 export const metadata = {
-  title: 'Create Next App',
-  description: 'Generated by create next app',
-}
+  title: "Beauty App",
+  description: "Beauty platform for managing and recommending beauty products.",
+};
```

Comment:
What happened here? I see all single-quotes replaced by double-quotes...

There should be no modifications in this repo except the title and description. If you want to do the other changes, commit them separately.